odoo/addons/base/models/assetsbundle.py

```diff
@@ -693,10 +783,27 @@ def minify(self):
     def with_header(self, content=None):
         if content is None:
             content = self.content
-        return '\n/* %s */\n%s' % (self.name, content)
+        return ('\n/* %s */\n%s' % (self.name, content))
```

Comment:
remove ()

Reply:
Also could use f-strings, since they're used elsewhere (and the expressions are trivial).